src/tiling/tile_state.rs

```diff
@@ -50,6 +50,8 @@ pub struct TileStateMut<'a, T: Pixel> {
   pub input_tile: Tile<'a, T>, // the current tile
   pub input_hres: &'a Plane<T>,
   pub input_qres: &'a Plane<T>,
+  pub activity_mask: &'a Plane<u16>,
+  pub mean_activity: u16,
```

Comment:
Do we want a mean activity per-tile or per frame or have it with a finer granularity?

Reply:
I don't think we need any mean tile activity stored since the activity mask is not normalized anymore. We can calculate the mean in any context when required.